Application/EdFi.Ods.Tests/EdFi.Ods.Common/Security/SecureHashAwareSecretVerifierTests.cs

```diff
@@ -4,212 +4,239 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 #if NETCOREAPP
+using System;
+using System.Collections.Generic;
 using EdFi.Ods.Common.Security;
 using EdFi.TestFixture;
 using FakeItEasy;
 using NUnit.Framework;
+using Shouldly;
 using Test.Common;
 
-namespace EdFi.Ods.Tests.EdFi.Ods.Sandbox.Security
+namespace EdFi.Ods.Tests.EdFi.Ods.Common.Security
 {
-    namespace EdFi.Ods.Tests.EdFi.Ods.Sandbox.Security
+    [TestFixture]
+    public class SecureHasAwareSecretVerifierTests
```

Comment:
Now that we've moved the responsibility for selecting the correct ISecureHasher into the SecureHashAwareSecretVerifier, we need additional tests to ensure its selecting the right implementation.  Unless we abstract that out as mentioned elsewhere.